website/content/en/blog/2024-05-30-cncf-cloud-native-sustainability-week-2024-is-underway-and-why-you-should-care.md

```diff
@@ -0,0 +1,77 @@
+* Virtual Sessions: Participate in webinars, workshops, and online discussions organized by the TAG ENV to explore topics such as renewable energy and IT resource optimization.  
+* Speak at an Event: Propose a session on sustainability topics at a local or online meetup. This is an opportunity to share your ideas and actively contribute to the sustainability discussions. 
+
+### A Look Back at CNCF Cloud Native Sustainability Week 2023
+
+In 2023, the CNCF Cloud Native Sustainability Week saw significant and diverse participation.
+
+Over 22 meetups were organized in more than 17 countries, involving numerous international speakers and thousands of participants. The events covered a wide range of topics, from renewable energy to IT resource optimization.
+
+Reflecting back on last year’s event, which was also organized for the first time, we see that it was a huge success! Based on the feedback from the community, it contributed to:
+
+* Increased Awareness: Awareness campaigns reached a broader audience, educating on sustainable practices through articles, social media posts, and webinars.  
+* Community Engagement: Active participation facilitated the sharing of knowledge and best practices, creating a network of sustainability supporters.  
+* Successful Initiatives: Specific projects were launched, such as improving IT infrastructures to reduce energy consumption.
+
+If you would like to relive the highlights of the Cloud Native Sustainability Week 2023 and hear some significant testimonials about the event, check out the [following video](https://youtu.be/ezGSYtvQH2c?si=3q-USUPyLVjBq9wH). This video gathers the experiences of participants and organizers, offering an inspiring look at the conducted activities and highlighting the most memorable moments from the event. We’ve also published a blog post wrapping up the CNCF Cloud Native Sustainability Week 2023: [Cloud Native Sustainability Week 2023 - putting sustainability on the community radar](https://tag-env-sustainability.cncf.io/blog/2024-wrap-up-cloud-native-sustainability-week-2023)
+
+## The CNCF Cloud Native Sustainability Week needs YOU!
+
+The success of the CNCF Sustainability Week relies on the contribution of passionate volunteers, and there are several ways you can help:
+
+Organizing Events: If you're willing, you can organize an event, support other local gatherings, or virtual sessions. This role is crucial to ensure each event runs smoothly and reaches its full potential.
+Sharing Your Expertise: If you have experience in sustainable practices or cloud native technologies, consider hosting a session or workshop. Sharing your knowledge can have a significant impact and inspire others to adopt sustainable practices.
+Active Participation: Even just attending events, promoting them within your network, and sharing your ideas can help create a more aware and engaged community. Every voice counts, and your contribution can make a difference.
```

Comment:
Suggestion: Can we make this is a list? the rendering puts it in a para and it is difficult to read.